tests/test_DateOfNextNewMoon.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+
+from control import getDateOfNextNewMoon
```

Comment:
Please add these tests (call the tests you created in test_DateOfNectNewMoon.py) to the test_all.py file, as we will run all unit tests at one time, not as individual files.
![image](https://user-images.githubusercontent.com/58247240/193940246-b872751c-6b6b-47ec-ad1f-1f2460505c2b.png)